src/hw_simulator.js

```diff
@@ -104,8 +104,27 @@ exports.readEeproms = function (eeproms) {
     return (eeproms);
 };
 
-exports.readPlatform = function (platform) {
+var readPlatform = function (platform) {
     winston.info('readPlatform()');
     platform.name = 'BeagleBone Simulator';
     return (platform);
-};
\ No newline at end of file
+};
+
+module.exports = {
+    logfile: logfile,
+    readPWMFreqAndValue: readPWMFreqAndValue,
+    readGPIODirection: readGPIODirection,
+    readPinMux: readPinMux,
+    setPinMode: setPinMode,
+    setLEDPinToGPIO: setLEDPinToGPIO,
+    exportGPIOControls: exportGPIOControls,
+    writeGPIOValue: writeGPIOValue,
+    readGPIOValue: readGPIOValue,
+    enableAIN: enableAIN,
+    readAIN: readAIN,
+    writeGPIOEdge: writeGPIOEdge,
+    writePWMFreqAndValue: writePWMFreqAndValue,
+    readEeproms: readEeproms,
+    readPlatform: readPlatform,
+
```

Comment:
Again, extraneous. The repetition makes me wonder if the definition of what gets exported from these hw_X.js files should be common as they satisfy an internal API.